pkg/ee/metering/cronjob.go

```diff
@@ -58,6 +58,8 @@ func cronJobReconciler(reportName string, mrc *kubermaticv1.MeteringReportConfig
 
 			job.Spec.Schedule = mrc.Schedule
 			job.Spec.JobTemplate.Spec.Parallelism = pointer.Int32(1)
+			job.Spec.JobTemplate.Spec.Template.Spec.ServiceAccountName = ""
+			job.Spec.JobTemplate.Spec.Template.Spec.DeprecatedServiceAccount = ""
```

Comment:
because `DeprecatedServiceAccount` value is autoset by kube-apiserver, have to unset that too.